### DISCUSSION TITLE
Listbox widget for customtkinter for multiple choice for the user

### DISCUSSION THREAD

**Discussion by saiyam-sandhir** [TomSchimansky/CustomTkinter#1225] [Ideas]
CustomTkinter only consists of OptionMenu and ComboBox widget but it would be great to have a ListBox widget so that the user can choose multiple options and the coder/designer can add a good-looking Listbox rather than the ugly default one

**Comment by jh-jensen** [comment]
I agree, have been making my own so far, but it's a struggle not being able to directly feed it a tupel value for light/dark theme which I currently update manually when the users toggle the switch.

**Comment by Akascape** [comment]
@saiyam-sandhir 
**Hi, I just made this quick listbox for customtkinter:**
https://github.com/Akascape/CTkListbox

![](https://user-images.githubusercontent.com/89206401/247524994-256c135f-3d6c-4daa-87a5-b4572a2c905f.jpg)
